README.md

```diff
@@ -328,18 +328,27 @@ curl -u "username:access_key" -X GET https://api.browserstack.com/5/workers
  
 Example response:
 ```javascript
-[{
- "id": "<workerId>",
- "status": "running",
- "os": "OS X",
- "os_version": "Mojave",
- "browser": "chrome",
- "browser_version": "75.0",
- "real_mobile": null,
- "device": null,
- "browser_url": "<dashboard_url_of_the_session>",
- "sessionId": "<sessionId>"
-}]
+[
+  {
+   "id": "<workerId>",
+   "status": "running",
+   "os": "OS X",
+   "os_version": "Mojave",
+   "browser": "chrome",
+   "browser_version": "75.0",
+   "real_mobile": null,
+   "device": null,
+   "browser_url": "<dashboard_url_of_the_session>",
+   "sessionId": "<sessionId>"
+  },
+ {
```

Comment:
Is this the response? If yes, maybe explicitly title it?

Reply:
yes, this is a response. I've title it under "Example response." Please check.